scripts/gff_metaparser/genmetaconf/manifest.py

```diff
@@ -24,70 +24,70 @@
 
 from os.path import dirname, join as pj
 
-class Manifest:
-  def __init__(self, mapping, ungzip = True, always_copy = True):
```

Comment:
don't get this diff

Reply:
Seems to me to be reformatting the level of indentation more than anything else